### PR TITLE
Issue #3024 - Companion 2.1 (windows) - Lua 'execute' fails in non-root directory

### PR DESCRIPTION
**Pull request by dsbeach**
~~Conversion of backward slash to forward slash for companion.~~

Fixes #3025 

Solution was to modify convertSimuPath() to append fully qualified path name for requests starting with both forward and backward slashes.

### CODE REVIEW

**Comment by projectkk2glider** [comment]
I think the change should be done in simpgmspace.cpp as it has nothing to do with the radio.

**Comment by dsbeach** [comment]
I'll investigate and see if I can find the right spot.

File: radio/src/targets/simu/simpgmspace.cpp [hunk 1]
```diff
@@ -1006,6 +1006,12 @@ FRESULT f_getcwd (TCHAR *path, UINT sz_path)
 
   // remove simuSdDirectory from the cwd
   strcpy(path, cwd + strlen(simuSdDirectory));
+
+  // convert backslashes to forward - issue #3024
+  for (int i = 0; path[i] != 0; i++) {
+	  if (path[i] == '\\') { path[i] = '/'; }
```
Comment (bsongis) [inline radio/src/targets/simu/simpgmspace.cpp]: Code style not ok!

Comment (dsbeach) [inline radio/src/targets/simu/simpgmspace.cpp]: Please elaborate. Is there a style guide?

Comment (bsongis) [inline radio/src/targets/simu/simpgmspace.cpp]: Have a look in other files, you will see, here the indentation is KO (we use 2 chars) and a new line is missing

Comment (projectkk2glider) [inline radio/src/targets/simu/simpgmspace.cpp]: I've started a style guide here https://github.com/opentx/opentx/wiki/OpenTX-Code-Style-Guide

**Pull request merged**